pkg/pillar/cmd/zedagent/handleconfig.go

```diff
@@ -499,6 +500,19 @@ func touchSavedConfig(filename string) {
 	}
 }
 
+// Check if SavedConfig exists
+func existsSavedConfig(filename string) bool {
+	filename = filepath.Join(checkpointDirname, filename)
+	_, err := os.Stat(filename)
+	if err != nil {
+		if os.IsNotExist(err) {
```

Comment:
Shouldn't this condition be reversed to !os.IsNotExist?

Reply:
Correct, fixed, thank you!